packages/aws-rfdk/lib/core/lib/mongodb-instance.ts

```diff
@@ -304,11 +304,27 @@ export interface IMongoDb extends IConnectable, IConstruct {
  * Resources Deployed
  * ------------------------
  * - {@link StaticPrivateIpServer} that hosts MongoDB.
- * - An A Record in the provided PrivateHostedZone to create a DNS entry for this server's static private IP.
- * - A Secret containing the administrator credentials for MongoDB.
- * - An encrypted EBS Volume on which the MongoDB data is stored.
+ * - An A-Record in the provided PrivateHostedZone to create a DNS entry for this server's static private IP.
+ * - A Secret in AWS SecretsManager that contains the administrator credentials for MongoDB.
+ * - An encrypted Amazon Elastic Block Store (EBS) Volume on which the MongoDB data is stored.
  * - Amazon CloudWatch log group that contains instance-launch and MongoDB application logs.
  *
+ * Security Considerations
+ * ------------------------
+ * - The administrator credentials for MongoDB are stored in a Secret within AWS SecretsManager. You must strictly limit
+ *   access to this secret to only entities that require it.
+ * - The instances deployed by this construct download and run scripts from your CDK bootstrap bucket when that instance
+ *   is launched. You must limit write access to your CDK bootstrap bucket to prevent an attacker from modifying the actions
+ *   performed by these scripts. We strongly recommend that you either enable Amazon S3 server access logging on your CDK
+ *   bootstrap bucket, or enable AWS CloudTrail on your account to assist in post-incident analysis of compromised production
+ *   environments.
+ * - The EBS Volume that is created by, or provided to, this construct is used to store the contents of your MongoDB data. To
```

Comment:
Should we have a note about encryption at rest on this volume?  at least if they provide us one

Reply:
Certainly doesn't hurt!